client/van_site_config_create.go

```diff
@@ -18,7 +18,18 @@ func (cli *VanClient) VanSiteConfigCreate(ctx context.Context, spec types.VanSit
 		ObjectMeta: metav1.ObjectMeta{
 			Name: "skupper-site",
 		},
-		Data: map[string]string{},
+		Data: map[string]string{
+			"name":                   types.DefaultVanName,
+			"edge":                   "false",
+			"service-controller":     "true",
+			"service-sync":           "true",
+			"console":                "false",
```

Comment:
I think the console should be on by default (though I am fine if we have to temporarily default to false for some reason).